client/src/components/layout/DestinationsIndex.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from "react";
+import DestinationTile from "./DestinationTile";
+
+const DestinationsIndex = (props) => {
+
+    const [destinations, setDestinations] = useState([])
+
+    const getDestinations = async () => {
+        try {
+            const response = await fetch("/api/v1/destinations")
+            if (!response.ok) {
+                const errorMessage = (`${response.status} ${response.statusText}`)
+                const error = new Error(errorMessage)
+                throw(error)
+            }
+            const body = await response.json()
+            const newDestinations = body.destinations
+            setDestinations(
+                ...destinations,
+                newDestinations
+            )
+        } catch (err) {
+            console.error(`Error in Fetch, ${err.message}`)
+        }
+    }
+
+    useEffect(() => {
+        getDestinations()
+    }, [])
+
+    const destinationTiles = destinations.map((destination) => {
+        return (
+            <p key={destination.id}>
```

Comment:
to better separate concerns I'd apply all formatting and styling for a destination in the DestinationTile. This `<p>` is also creating awkward HTML hierarchy where for every destination we create a p tag and within it is a `div` (defined in the DestinationTile)
```html
<p>
    <div>
        ...
    </div>
</p>
```
You can instead keep everything the same in the DestinationTile, remove the `p` here in the `map` return entirely, and move the `key` to the DestinationTile as a prop as it will be the top level/ only thing returned from this `map`
```js
return (
    <DestinationTile key={destination.id} destination={destination} />
)
```